using_images/other_images/jenkins.adoc

```diff
@@ -86,44 +80,31 @@ The Jenkins password can be configured with the following environment variable:
 
 
 === Volume Mount Points
-The Jenkins image can be run with mounted volumes to enable persistent storage for
-the configuration:
+The Jenkins image can be run with mounted volumes to enable persistent storage for the configuration:
 
-* *_/var/lib/jenkins_* - This is the data directory where Jenkins stores
-configuration files including job definitions.
+* *_/var/lib/jenkins_* - This is the data directory where Jenkins stores configuration files including job definitions.
 
 == Creating a Jenkins Service from a Template
 
-OpenShift provides a link:../../dev_guide/templates.html[template] to make
-creating a new Jenkins service easy. The template provides parameter fields to
-define all the environment variables (password) with predefined defaults. It
-will also define both a
-link:../../architecture/core_concepts/deployments.html#deployments-and-deployment-configurations[deployment
-configuration] and a
-link:../../architecture/core_concepts/pods_and_services.html#services[service].
-
-The Jenkins templates should have been registered in the default *openshift*
-project by your cluster administrator during the
-link:../../install_config/install/first_steps.html[First Steps] setup process.
-There are two templates available:
-
-* `jenkins-ephemeral` is for development or testing purposes only because it uses
-ephemeral storage for the Jenkins content. This means that if the Jenkins
-pod is restarted for any reason, such as the pod being moved to another node
-or the deployment configuration being updated and triggering a redeploy, all
-data will be lost.
-* `jenkins-persistent` uses a persistent volume store for the Jenkins content which
-means the data will survive a pod restart. Using persistent volumes requires a
-persistent volume pool be defined in the OpenShift deployment. Cluster
-administrator instructions for setting up the pool are located
-link:../../admin_guide/persistent_storage/persistent_storage_nfs.html[here].
-
-You can find instructions for instantiating templates by following these
-link:../../dev_guide/templates.html#creating-resources-from-a-template[instructions].
-
-== Using Jenkins
-The OpenShift Jenkins image provides a pre-defined job which implements a very basic build/test/promote flow.
-
-The job will trigger a new build of a `*BuildConfig*` defined in OpenShift, test the output of that build,
-and then retag the output to indicate it is ready for production.  For more details on this sample job, see
-this link:https://github.com/openshift/origin/blob/master/examples/jenkins/README.md[tutorial].
+link:../../dev_guide/templates.html[Templates] provide parameter fields to define all the environment variables (password) with predefined defaults.
```

Comment:
@tnguyen-rh yup, you're right and that's better.  I think of it as one template because you use one or the other, you'd never use both and they're basically the same except for the volume type, but you're absolutely correct.